tests/mocks/metricsRecorder.ts

```diff
@@ -0,0 +1,41 @@
+import { expect, sinon } from '@loopback/testlab'
+import { Pool } from 'pg'
+import rewiremock from 'rewiremock'
+import { InfluxDB, WriteApi } from '@influxdata/influxdb-client'
+import { MetricsRecorder } from '../../src/services/metrics-recorder'
+import { Redis } from 'ioredis'
```

Comment:
Let's move this import upwards so we only have relative paths below the `influx` import.